src/api/app/services/notification_service/notifier.rb

```diff
@@ -30,6 +30,7 @@ def call
       end
     rescue StandardError => e
       Airbrake.notify(e, event_id: @event.id)
+      raise e unless Rails.env.production?
```

Comment:
This all makes little sense. Why catch and report this at all?

Reply:
Do you think we can remove this error handler? I do not remember why we put this there, this change was introduced 2 years ago... This is used by Clockwork, that has its own error handler, and it is also used in a data migration, which will report errors on the stderr...